web/impact/impact/tests/test_mentor_program_office_hour_list_view.py

```diff
@@ -0,0 +1,164 @@
+# MIT License
+# Copyright (c) 2019 MassChallenge, Inc.
+
+import json
+from jsonschema import Draft4Validator
+
+from django.urls import reverse
+
+from accelerator.tests.factories import (
+    EntrepreneurFactory,
+    ExpertFactory,
+)
+
+from impact.tests.factories import MentorProgramOfficeHourFactory
+from impact.tests.api_test_case import APITestCase
+from impact.v1.helpers.mentor_program_office_hour_helper import (
+    OFFFICE_HOUR_FIELDS,
+)
+from impact.tests.utils import assert_fields
+from impact.v1.views import MentorProgramOfficeHourListView
+
+USER_COUNT = 3
+NON_USER_COUNT = 4
+NON_EXISTENT_ID = 999
+NON_EXISTENT_NAME = 'qwerty'
+
+
+class TestMentorProgramOfficeHourListView(APITestCase):
+    url = reverse(MentorProgramOfficeHourListView.view_name)
+
+    def setUp(self):
+        self.test_office_hours = MentorProgramOfficeHourFactory.create_batch(
+            NON_USER_COUNT
+        )
+
+    def tearDown(self):
+        map(lambda obj: obj.delete(), self.test_office_hours)
+
+    def test_get(self):
+        _, office_hours = self._create_user_office_hours()
+        response = self._get_response_as_logged_in_user()
+        total_count = USER_COUNT + NON_USER_COUNT
+        self.assertEqual(response.data["count"], total_count)
+        self.assertTrue(self._compare_ids(office_hours, response))
+        self.assertTrue(self._compare_ids(self.test_office_hours, response))
+
+    def test_options(self):
```

Comment:
this test name could be better, where testing that the options do what? and which options?